specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2023-04-01-preview/costmanagement.exports.json

```diff
@@ -416,6 +416,18 @@
         }
       ],
       "properties": {
+        "identity": {
+          "description": "The managed identity associated with Export",
+          "$ref": "common-types.json#/definitions/IdentityMetadata"
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the Export's managed identity. Only required when utilizing managed identity.",
+          "x-ms-mutability": [
+            "create",
+            "read"
```

Comment:
What is this locaiton for ? Location is not really in the official documentation for managed identity so what is this used for ?

Reply:
As per this link https://eng.ms/docs/products/arm/rbac/managed_identities/msionboardingproxyresource, ARM uses location for MIRP routing. See below.
![image](https://github.com/Azure/azure-rest-api-specs/assets/118343862/46e129a2-9982-4e2b-9481-2a803a07584d)